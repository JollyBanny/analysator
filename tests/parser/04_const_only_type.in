const constant: integer;

begin
end.