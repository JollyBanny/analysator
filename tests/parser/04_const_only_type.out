[Line: 1; Column: 24]	'=' expected but ';' found
